Submitted By: Mario Fetka (geos_one) (mario dot fetka at gmail dot com)
Date: 2009-12-20
Initial Package Version: 1.2.2
Origin: me
Upstream Status: never accepted
Description: change search patch for svrcore headers
 
diff -Naur freeipa-1.2.2.orig/ipa-server/configure.ac freeipa-1.2.2/ipa-server/configure.ac
--- freeipa-1.2.2.orig/ipa-server/configure.ac	2009-12-20 15:05:57.119860118 +0000
+++ freeipa-1.2.2/ipa-server/configure.ac	2009-12-20 15:07:03.622615066 +0000
@@ -124,7 +124,7 @@
 dnl will not build against OpenLDAP.
 SAVE_CPPFLAGS=$CPPFLAGS
 CPPFLAGS="-I/usr/include/nspr -I/usr/include/nss"
-AC_CHECK_HEADER(svrcore.h)
+AC_CHECK_HEADER(svrcore/svrcore.h)
 if test "x$ac_cv_header_svrcore_h" = "xno" ; then
 	AC_MSG_ERROR([Required svrcore header not available (svrcore-devel)])
 fi
